### ISSUE TITLE
Remove semicolon before JSX

### ISSUE THREAD

**Issue by gaearon** [ritz078/transform#319]
I find it very confusing that HTML gets transformed with JSX with `;` prepended to the output. Since people are going to copy this, this introduces a hazard like

```js
return ;<>
  {/* Hello world */}
  <div className="awesome" style={{ border: "1px solid red" }}>
    <label htmlFor="name">Enter your name: </label>
    <input type="text" id="name" />
  </div>
  <p>Enter your HTML here</p>
</>
```

which returns nothing.

Can we remove this? Thanks!

**Comment by NaincyKumariKnoldus** [comment]
Hey @gaearon I have raised a PR now for this. Please review my PR!

**Comment by ritz078** [comment]
@NaincyKumariKnoldus Your PR didn't fix the actual issue. The result still included a `;` at the start.
  
@gaearon I have merged fix for this and #279
